packages/react-tinacms-github/README.md

```diff
@@ -127,6 +127,10 @@ const cms = new TinaCMS({
 })
 ```
 
+### Image formats for preview
+
+Currently, the media manager and github media store only supports previews for image files of the following formats: '.jpg', '.jpeg', '.png', '.webp', '.svg'. Other file types will show a file icon instead of an image preview in the media manager. 
```

Comment:
```suggestion
Currently, the media manager and GitHub media store only supports previews for image files of the following formats: '.jpg', '.jpeg', '.png', '.webp', '.svg'. Other file types will show a file icon instead of an image preview in the media manager. 
```